### PR TITLE
fix(map/basic): fix map basic hover problems with markers and polygons

### PR DESCRIPTION
**Pull request by midudev**
Fix hover with map basic

### CODE REVIEW

File: components/map/basic/src/leaflet/shapes/Polygons.js [hunk 1]
```diff
@@ -51,17 +53,43 @@ export default class SearchMapPolygons {
...
+            // remove previous hovered polygons
+            const polygonWithHover = document.querySelector(
+              `.${that.HOVER_CLASSNAME}`
+            )
+            // if there's a previous polygon, remove the hover class
+            polygonWithHover &&
+              polygonWithHover.classList.remove(that.HOVER_CLASSNAME)
+
+            // add the hover classname to the new polygon
+            debugger //eslint-disable-line
```
Comment (midudev) [inline components/map/basic/src/leaflet/shapes/Polygons.js]: Remove the debichos.

Comment (ruben-martin-lozano) [inline components/map/basic/src/leaflet/shapes/Polygons.js]: oops, debugger

Comment (juanmendez90) [reply components/map/basic/src/leaflet/shapes/Polygons.js]: done

Comment (juanmendez90) [reply components/map/basic/src/leaflet/shapes/Polygons.js]: removed

**Pull request merged**